docs/cellery-syntax.md

```diff
@@ -63,11 +63,10 @@ cellery:Component helloComponent = {
     }
 };
 ```
-
 #### Ingresses
 An Ingress represents an entry point into a cell component. An ingress can be an HTTP, TCP, GRPC or Web endpoint.
 
-##### 1. HTTP Ingress
+##### 1. HTTP Ingresses
```

Comment:
These are linked to the content list of this readme that is defined in the beginning. With this change, those links may not be working. Please fix accordingly.